code/food.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="food.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<header>
+    <menu>
+        <ul class="horizontal-menu">
+            <a href="home.html"> <i class="fa fa-envira" style="font-size:40px;color: #2f2e2e"></i></a>
+             <li> <a href="food.html"> food </a> </li>
+             <li> <a href="nature.html"> nature </a> </li>
+             <li> <a href="interests.html"> interests </a> </li>
+             <li> <a href="metime.html"> me time</a> </li>
+           </ul>
+    </menu>
+</header>
+<body>
+    <div class="boxes">
+        <div class="content-box has-featured-img">
+          <img class="featured-img" src="coffee.jpg" alt="coffee">
```

Comment:
When an HTML element has several attributes, it's good for readability to give them new lines:
```
<img
  class="featured-img"
  src="coffee.jpg"
  alt="coffee">
```